ginac/symbol.cpp

```diff
@@ -171,7 +171,7 @@ void symbol::do_print_python_repr(const print_python_repr & c, unsigned level) c
 void symbol::set_domain(unsigned d)
 {
         domain = d;
-        iflags.clear();
+        //iflags.clear();
```

Comment:
In future please simply delete and leave no outcommented code. It only adds to any confusion.